packages/big-design/src/components/Dropdown/spec.tsx

```diff
@@ -505,3 +505,29 @@ test('clicking label does not call onItemClick', async () => {
 
   await waitForElement(() => screen.getByRole('option', { name: /option 1/i }));
 });
+
+test('renders appropriate amount of list items', async () => {
+  const { getByRole, container } = render(
+    <Dropdown
+      items={[
+        {
+          label: 'Label 1',
+          items: [{ content: 'Option 1', onItemClick }],
+        },
+        {
+          items: [{ content: 'Option 2', onItemClick }],
+        },
+      ]}
+      toggle={<Button>Button</Button>}
+    />,
+  );
+  const toggle = getByRole('button');
+
+  await act(async () => {
```

Comment:
You shouldn't need to `act` here or `await` for fireEvent.

Maybe just wait for an item like:
```js
await waitForElement(() => screen.getByRole('option', { name: /option 1/i }));
```